### PR TITLE
Issue 405

### LINKED ISSUE #405: Backoffice - Add indication of passengers / packages
To do:
1. Add indication in rides table
2. Add picklist in create / edit ride dialog

Table Figma:
https://www.figma.com/file/3UBN2ek01pmm2muttlrxYj/MVP-Current-Design?type=design&node-id=1549-163367&mode=dev

Dialog Figma:
https://www.figma.com/file/3UBN2ek01pmm2muttlrxYj/MVP-Current-Design?type=design&node-id=1301-331406&mode=dev

### PR DESCRIPTION
**Pull request by aviv-taori**
closes #405

### CODE REVIEW

File: client/src/Backoffice/components/Main/components/Rides/Rides.tsx [hunk 1]
```diff
@@ -80,8 +82,26 @@ const columns: ColumnDef<Partial<Ride>>[] = [
   },
   {
     accessorKey: 'passengerCount',
-    header: "מס' נוסעים",
-    accessorFn: (data) => data.passengerCount || '-'
+    header: 'נוסעים/ ארגזים',
+    cell: ({ row }) => {
+      console.log(row.original.passengerCount);
```
Comment (LEVI-RIVKIN) [inline client/src/Backoffice/components/Main/components/Rides/Rides.tsx:87]: @aviv-taori Remove `console.log`

File: client/src/Backoffice/components/Main/components/Rides/Rides.tsx [hunk 2]
```diff
@@ -80,8 +82,26 @@ const columns: ColumnDef<Partial<Ride>>[] = [
...
+
+      if (!row.original.passengerCount) return <span>'-'</span>;
+
+      if (row.original.serviceType === 'Ride') {
```
Comment (LEVI-RIVKIN) [inline client/src/Backoffice/components/Main/components/Rides/Rides.tsx:91]: @aviv-taori Use the enum as discussed

File: client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx [hunk 3]
```diff
@@ -22,6 +27,12 @@ function NewRideInfo() {
   const specialRequestsDefaultValue = DRIVER_CAPABILITIES.map(({ value }) => value);
   const selectedSpecialRequests = watch('specialRequest', specialRequestsDefaultValue) || [];
 
+  const [serviceType, setServiceType] = React.useState('Ride');
```
Comment (LEVI-RIVKIN) [inline client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx:30]: @aviv-taori Should use the form ride data, no need for a separate state

File: openapi.yaml [hunk 4]
```diff
@@ -550,8 +550,9 @@ components:
         serviceType:
           type: string
           enum:
-            - ride
-            - delivery
+
```
Comment (LEVI-RIVKIN) [inline openapi.yaml:553]: @aviv-taori No need for the empty line

**Review by LEVI-RIVKIN — CHANGES_REQUESTED** [review]

File: client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx (hunk 3 above)
Comment (aviv-taori) [reply client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx:30]: Needed for default. No?

File: client/src/Backoffice/components/Main/components/Rides/Rides.tsx (hunk 1 above)
Comment (aviv-taori) [reply client/src/Backoffice/components/Main/components/Rides/Rides.tsx:87]: I deleted

File: client/src/Backoffice/components/Main/components/Rides/Rides.tsx (hunk 2 above)
Comment (aviv-taori) [reply client/src/Backoffice/components/Main/components/Rides/Rides.tsx:91]: Done

File: openapi.yaml (hunk 4 above)
Comment (aviv-taori) [reply openapi.yaml:553]: I deleted

File: client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx (hunk 3 above)
Comment (LEVI-RIVKIN) [reply client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx]: @aviv-taori You can set the default in the `useForm`

**Comment by LEVI-RIVKIN** [comment]
@aviv-taori Go over the `verify` comments as well..

File: client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx (hunk 3 above)
Comment (aviv-taori) [reply client/src/Backoffice/components/Main/components/modals/AddRide/NewRideInfo/NewRideInfo.tsx]: Ok!

**Review by LEVI-RIVKIN — APPROVED** [review]

**Pull request merged**